### ISSUE TITLE
Cannot reproduce the results of BART on ELI5

### ISSUE THREAD

**Issue by 311dada** [pytorch/fairseq#3504]
#### What have you tried?
First, I followed [ELI5](https://github.com/facebookresearch/ELI5) and constructed the dataset. Then, I finetuned BART on the dataset. Specifically, the question and documents are concatenated as the input of the encoder and the answer is the output of the decoder.

**Comment by mengyanggithub** [comment]
Can you reproduce the results in the end? @311dada

**Comment by mengyanggithub** [comment]
Did you solve this problem? @311dada
